### DISCUSSION TITLE
Oblivious DNS over HTTPS

### DISCUSSION THREAD

**Discussion by Delkaloop** [trinib/Adguard-Wireguard-Unbound-Cloudflare#22] [General]
What do you think about adding oDoH to this configuration?

**Comment by trinib** [comment]
This looks promising for the future..oDoH is a new network technology came out in 2019.. I'll look more into it and see if and how it works

**Reply by Delkaloop** [reply]
I'm looking forward to it

**Comment by trinib** [comment]
@Delkaloop created guide for Oblivious DNS over HTTPS: https://github.com/trinib/AdGuard-WireGuard-Unbound-Cloudflare/wiki/Install-DNScrypt-proxy-(DoH)(oDoH)(Anonymized-DNS)
